Parsers/ASimNetworkSession/Tests/VMwareCarbonBlackCloud_ASimNetworkSession_DataTest.csv

```diff
@@ -0,0 +1,16 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10262722 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Carbon Black Cloud""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10262722 records (100.0%) for field [EventVendor] of type [Enumerated]: [""VMware""] (Schema:NetworkSession)"
```

Comment:
Can you add Carbon Black, VMware details in EventProduct and EventVendor in this file ?https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/dev/ASimTester/ASimTester.csv

Reply:
Done